### PR TITLE
Configure Carthage compatibility with Xcodegen

### PR DESCRIPTION
**Pull request by joeydong**

### CODE REVIEW

**Comment by joeydong** [comment]
👀 @marcelofabri ?

**Comment by joeydong** [comment]
I've already created a branch in our main app to point to this branch and it compiles, runs, and behaves as expected – unchanged!

File: project.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+targets:
+  FontAwesome:
+    platform: iOS
+    scheme: {} # Generate default scheme
+    settings:
+      GCC_GENERATE_DEBUGGING_SYMBOLS: NO # Avoid downstream warning: "Redistributable static libraries should never be built with module debugging enabled"
+      MACH_O_TYPE: staticlib
+      MODULEMAP_FILE: FontAwesome/FontAwesome.modulemap
+    sources:
+      - path: FontAwesome
```
Comment (marcelofabri) [inline project.yml:15]: shouldn't this handle the font file somehow? 🤔

Comment (joeydong) [reply project.yml:15]: so long story short, our app already has a copy of `FontAwesome.ttf`. I tried removing that copy and installing this as a dynamic embedded framework but the font wasn't found when I manually tested the app. I suspect there's some stipulation like fonts need to be copied by the main app. Therefore I decided to keep the main app copy and just make this a static framework!

Comment (marcelofabri) [reply project.yml:15]: 🧐

**Review by marcelofabri — APPROVED** [review]

**Pull request merged**